source/appModules/explorer.py

```diff
@@ -189,6 +191,19 @@ def _get_windowText(self):
 			return windowText.replace(CHAR_LTR_MARK,'').replace(CHAR_RTL_MARK,'')
 		return windowText
 
+
+class MetadataEditField(RichEdit50):
+	""" Used for metadata edit fields in Windows Explorer in Windows 7.
+	By default these fields would use ITextDocumentTextInfo ,
+	but to avoid Windows Explorer crashes we need to use EditTextInfo here. """
+	@classmethod
+	def _get_TextInfo(cls):
+		if ((winVersion.winVersion.major, winVersion.winVersion.minor) == (6, 1)):
+			cls.TextInfo = EditTextInfo
+		else:
+			cls.TextInfo = super(MetadataEditField, self).TextInfo

```

Comment:
Ugh. I made this mistake in my initial comment and updated the comment afterwards. Yet, the initial version of the comment ended up in the code:

```suggestion
			cls.TextInfo = super(MetadataEditField, cls).TextInfo
```